### DISCUSSION TITLE
What prompts were used for the examples?

### DISCUSSION THREAD

**Discussion by johnmclane99** [SupaGruen/SD-CheatSheet#19] [General]
Curious what the exact prompts were for each of the four example images for each style.  #2 and #3 are obviously woman and man, but what are the exact prompts?  Thank you; this is an impressive guide that I frequently use!

**Comment by SupaGruen** [comment]
Hi @johnmclane99, I wrote a bit about the prompts/setup here:
https://supagruen.github.io/StableDiffusion-CheatSheet/documentation/index.html
Thank you, I'm glad it's useful. All the best!
